lighthouse-core/test/chromium-web-tests/README.md

```diff
@@ -21,15 +21,12 @@ SKIP_DOWNLOADS=1 yarn test-devtools
 ### Prerequistes
 
 <details>
-  <summary> Install `requests`</summary>
-
-  Ensure you have `requests` module available globally on your python 2.7 install. New Macs do not come with pip for python 2.7 which is deprecated, so you might have to install that too.
+  <summary>Install python dependencies</summary>
 
+  Ensure you have these modules available globally on your python 3.x install:
 
   ```sh
-  curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
-  python get-pip.py
-  pip -m pip install requests
+  python3 -m pip install requests six
```

Comment:
Do we need to use this command in CI now?

Reply:
This isn't new, just wasn't documented. CI still fine

Reply:
I mean, CI is still complaining about the missing `six` module. This was my best guess as to why.

Reply:
Oh weird I was just looking at it said it passed. Gh lying to me